keyboards/handwired/108key_trackpoint/README.md

```diff
@@ -0,0 +1,23 @@
+# 108 Key with Trackpoint
+
+![wiring](wiring.JPG)
```

Comment:
You need to update the URL part of this link now.

Reply:
Yes, I do, I'm about to PR the qmk.mf repo. I'll tag you once I have and I've made a couple of other small changes. Thanks for all your help and time so far

Reply:
That PR has been merged, so you should be good to modify this now.